xml/System.Security.Cryptography/AesManaged.xml

```diff
@@ -105,9 +105,13 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the block size, in bits, of the cryptographic operation.</summary>
+        <value>The block size, in bits, of the cryptographic operation. The default is 128 bits.</value>
+        <remarks><format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The block size is the basic unit of data that can be encrypted or decrypted in one operation. Messages longer than the block size are handled as successive blocks; messages shorter than the block size must be padded with extra bits to reach the size of a block. For AES, the only valid block size is 128 bits.    
```

Comment:
Seems like this should copy, or point to, SymmetricAlgorithm's definition of this.